src/server/scripts/Northrend/zone_borean_tundra.cpp

```diff
@@ -2384,6 +2384,108 @@ class spell_windsoul_totem_aura : public SpellScriptLoader
     }
 };
 
+enum BloodsporeRuination
+{
+    NPC_BLOODMAGE_LAURITH = 25381,
+    TALK_0 = 0,
```

Comment:
Because the word `Talk` is used already in the script as the action, use something beginning with `SAY_` to signify that the referenced text is said by the NPC. Use something descriptive like `SAY_BLOODMAGE_LAURITH` or use something directly related to the event or naming the event (`SAY_BLOODSPORE_RUINATION_TEST`), especially if the NPC in question has got many Say lines for different occasions (if the text said by the NPC is aggro, player kill or own death, it is usually named `SAY_AGGRO` `SAY_SLAY` or `SAY_DEATH` which is common in boss scripts).

Cosmetic, but worth noting: the old standard for creature_text comments when the text was used in a C++ script was to include the enum name as either part of the `creature_text`.`comment` field in DB or using the enum name as the only comment content.